plugin/helpers_bots_test.go

```diff
@@ -447,4 +447,28 @@ func TestShouldProcessMessage(t *testing.T) {
 
 		assert.True(t, shouldProcessMessage)
 	})
+
+	t.Run("should not process the message which have from_webhook", func(t *testing.T) {
+		channelID := "1"
+		api := setupAPI()
+		api.On("GetChannel", channelID).Return(&model.Channel{Id: channelID, Type: model.CHANNEL_GROUP}, nil)
+		p.API = api
+		api.On("KVGet", plugin.BOT_USER_KEY).Return([]byte(expectedBotId), nil)
+
+		shouldProcessMessage, _ := p.ShouldProcessMessage(&model.Post{ChannelId: channelID, Props: model.StringInterface{"from_webhook": true}}, plugin.AllowBots())
+
+		assert.False(t, shouldProcessMessage)
+	})
+
+	t.Run("should process the message which have from_webhook with allow webhook plugin", func(t *testing.T) {
+		channelID := "1"
+		api := setupAPI()
+		api.On("GetChannel", channelID).Return(&model.Channel{Id: channelID, Type: model.CHANNEL_GROUP}, nil)
+		p.API = api
+		api.On("KVGet", plugin.BOT_USER_KEY).Return([]byte(expectedBotId), nil)
+
+		shouldProcessMessage, _ := p.ShouldProcessMessage(&model.Post{ChannelId: channelID, Props: model.StringInterface{"from_webhook": true}}, plugin.AllowBots(), plugin.AllowWebhook())
+
+		assert.True(t, shouldProcessMessage)
+	})
```

Comment:
Would you also please add a check for a post where `"from_webhook"` is not set and one where `"from_webhook"` is `false`?